.github/workflows/release.yml

```diff
@@ -28,4 +33,13 @@ jobs:
     - name: Publish to Github Container Registry
       run: |
         docker push docker.pkg.github.com/shabados/viewer-frontend:${GITHUB_SHA}
+        docker tag docker.pkg.github.com/shabados/viewer-frontend:${GITHUB_SHA} docker.pkg.github.com/shabados/viewer-frontend:latest
```

Comment:
I don't believe you need to tag `:latest`

Reply:
Just for making it simpler to show the latest one. I can remove it if you want.